repos/spack_repo/builtin/packages/paraview/package.py

```diff
@@ -287,6 +287,7 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("libtheora")
     depends_on("libtiff")
     depends_on("netcdf-c")
+    depends_on("parallel-netcdf", when="+netcdf-c+mpi")
```

Comment:
```suggestion
    depends_on("netcdf-c", when="~mpi")
    depends_on("parallel-netcdf", when="+mpi")
```

Reply:
netcdf-c depends on parallel-netcdf. are you sure?

Reply:
something like

depends_on("netcdf-c", when="~mpi")
depends_on("netcdf-c+parallel-netcdf", when="+mpi")